apps/meteor/client/components/AdministrationList/AdministrationList.tsx

```diff
@@ -1,41 +1,54 @@
 import { OptionDivider } from '@rocket.chat/fuselage';
-import type { FC } from 'react';
+import { useAtLeastOnePermission, usePermission } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
 import React, { Fragment } from 'react';
 
 import type { AccountBoxItem, IAppAccountBoxItem } from '../../../app/ui-utils/client/lib/AccountBox';
 import { isAppAccountBoxItem } from '../../../app/ui-utils/client/lib/AccountBox';
+import { useHasLicenseModule } from '../../../ee/client/hooks/useHasLicenseModule';
 import AdministrationModelList from './AdministrationModelList';
 import AppsModelList from './AppsModelList';
 import AuditModelList from './AuditModelList';
 
 type AdministrationListProps = {
 	accountBoxItems: (IAppAccountBoxItem | AccountBoxItem)[];
 	onDismiss: () => void;
-	hasAdminPermission: boolean;
-	hasAuditLicense: boolean;
-	hasAuditPermission: boolean;
-	hasAuditLogPermission: boolean;
-	hasManageApps: boolean;
 };
 
-const AdministrationList: FC<AdministrationListProps> = ({
-	accountBoxItems,
-	hasAuditPermission,
-	hasAuditLogPermission,
-	hasManageApps,
-	hasAdminPermission,
-	onDismiss,
-}) => {
+const ADMIN_PERMISSIONS = [
+	'view-logs',
+	'manage-emoji',
+	'manage-sounds',
+	'view-statistics',
+	'manage-oauth-apps',
+	'view-privileged-setting',
+	'manage-selected-settings',
+	'view-room-administration',
+	'view-user-administration',
+	'access-setting-permissions',
+	'manage-outgoing-integrations',
+	'manage-incoming-integrations',
+	'manage-own-outgoing-integrations',
+	'manage-own-incoming-integrations',
+	'view-engagement-dashboard',
+];
+
+const AdministrationList = ({ accountBoxItems, onDismiss }: AdministrationListProps): ReactElement => {
+	const hasAuditLicense = Boolean(useHasLicenseModule('auditing'));
```

Comment:
Won't this make the `loading` state become `true`?

Reply:
@ggazzo